tron/core/jobrun.py

```diff
@@ -308,6 +320,8 @@ def state(self):
             return ActionRun.STARTING
         if self.action_runs.is_failed:
             return ActionRun.FAILED
+        if self.seconds_until_run_time() == 0 and self.action_runs.is_blocked_on_trigger:
+            return ActionRun.RUNNING
```

Comment:
this seems kind of confusing - we want it to be marked running when it isn't?

Reply:
a lot of tron machinery depends on having correct states at correct times, i found this to be least confusing and smallest change that makes things work. we don't have "waiting on requirements" state because root actions couldn't have requirements in tron before - they would never start, now this assumption is broken.